### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by mykyta01**
[DEMO LINK](https://mykyta01.github.io/layout_moyo-header/)
 [TEST REPORT LINK](https://mykyta01.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1,3 +1,111 @@
...
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo-img {
+  margin: 0 0 0 50px;
+}
```
Comment (Andriy-Shymkiv) [inline src/style.css:15]: better to set `padding: 0 50px` for header or nav instead of this

File: src/style.css [hunk 2]
```diff
@@ -1,3 +1,111 @@
...
+.nav {
+  justify-content: center;
+
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0 50px 0 0;
```
Comment (Andriy-Shymkiv) [inline src/style.css:26]: again, set padding for header or nav instead

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,111 @@
...
+}
+
+.nav__item {
+  margin: 0 10px;
```
Comment (Andriy-Shymkiv) [inline src/style.css:30]: you can just `margin-right: 10px`

File: src/style.css [hunk 4]
```diff
@@ -1,3 +1,111 @@
...
+  padding: 23px 0;
```
Comment (Andriy-Shymkiv) [inline src/style.css:31]: come on, it's easier to center elements using flexbox

File: src/style.css [hunk 5]
```diff
@@ -1,3 +1,111 @@
...
+
+}
+
+.nav__link {
+  color: black;
+  display: block;
+  position: relative;
+  z-index: 1;
+  padding: 1.5em;
+  margin: -1.5em;
```
Comment (Andriy-Shymkiv) [inline src/style.css:41]: looks redunand, please remove this

File: src/style.css [hunk 6]
```diff
@@ -1,3 +1,111 @@
...
+  text-decoration: underline;
+  position: absolute;
+  bottom: 0;
+  width: 50%;
+  content: "";
+  display: block;
+  border: 2px solid var(--active-color);
+  border-radius: 8px;
+}
+/* idk how to do this @media part*/
```
Comment (Andriy-Shymkiv) [inline src/style.css:63]: you can remove this `@media` and try to pass the tests without it

File: src/style.css [hunk 7]
```diff
@@ -1,3 +1,111 @@
...
+}
+
+.logo-img {
+  margin: 0 0 0 50px;
+}
+
+.nav {
+  justify-content: center;
+
+}
```
Comment (Andriy-Shymkiv) [inline src/style.css:20]: you cant use `justify-content` if this block is not a flexbox, so please remove it

**Review by Andriy-Shymkiv — CHANGES_REQUESTED** [review]
please provide a working demo link in PR description

**Comment by mykyta01** [comment]
@nikitaromanishyn
[add task solution 2](https://github.com/mate-academy/layout_moyo-header/pull/3500/commits/d77b88be7e5186a7d820613b1e29e1d3dbaf17f5) 
here its finally works)

**Pull request closed without merging**